src/DiskIO/DiskDaemon/DiskdAction.cc

```diff
@@ -65,13 +65,13 @@ DiskdAction::Create(const Mgr::CommandPointer &aCmd)
 DiskdAction::DiskdAction(const Mgr::CommandPointer &aCmd):
     Action(aCmd), data()
 {
-    debugs(79, 5, HERE);
+    debugs(79, 5, "");
```

Comment:
> These are things where the HERE removal collides with other cleanup changes necessary

There are debugs() lines that need multiple changes. This PR is about the changes that (a) remove HERE and (b) can be easily automated. Everything else is out of this PR scope _and_ needs to wait for another manual line modification (or another script).

There is no "collision" here. Just multiple changes that should be done to the same line. Each change has its own set of preconditions. The preconditions for the changes in this PR are (a) HERE presence and (b) simple automation. The preconditions for changes like adding `ERROR` and `FATAL` prefixes are very different, and often include yet-undecided policy agreements. 

Those out-of-scope changes will be done later/separately. They will be done automatically in one pass (like this PR) or slowly/manually (like we have been polishing debugs() lines so far).

> and make the 100% scripted not possible for those lines

Divide and conquer:

- Removal of HERE can be 100% scripted. I believe I have proven that. Moreover, I have argued that scripting is the _only_ way we can remove all HEREs at once (because scripting allows everybody to remove HEREs from their branches which significantly reduces cross-branch porting overheads). 
- Some other changes may not be possible to script. I have never claimed that everything can be scripted. I do not understand why you seem to insist that each debugs() line can only be modified once. I can find no reasonable explanation why somebody might want to insist on that, especially when it is obvious that not everything can be scripted and it is (or should be) clear that scripting is key to accepting huge code changes that have little or no runtime effect.

> The script needs to be able to be integrated with something that does those other changes.

Why does it need to be? I see no such need. In fact, I see much _harm_ from doing that because it would be impractical to reproduce those manual changes across many branches. If somebody posts a PR that _manually_ modifies lots of debugs() lines just to polish them, I will probably vote it down because it creates too much pain for branch management. Either automate or wait for essential changes.

> So at worst skip the files where it cannot handle the necessary changes

That approach does not accomplish automation goals. The reason we want HERE removal automated (with a reusable script that everybody can apply to their branches as needed) is so that others can quickly repeat our changes. If we skip some files and then fix them manually, then others will not be able to fix their branches quickly. They would have to repeat all the manual steps we have done, which is a lot of pain (for little gain!).

> The intended workflow is that branch authors can merge/rebase the master branch changes into theirs

This is not a good workflow. Many branches do not follow master but still want to take patches/features from master. A script makes HERE removal trivial for such branches. Lots of manual changes create a significant overhead (while gaining little).

> All the cleanup polishing has the same problems impacting each other so calling one out of scope for the others is not going to avoid the problem

Scripting avoids most problems associated with HERE removal. Other manual polishing touches piggybacked to other necessary changes are not a problem. In combination, the approach improves code without creating significant pain. Does it solve all the problems? No, it does not, but solving all problems is impossible, so we must lower the bar if we want to speed up progress.

> We can omit them from the script

If HERE removal is not automated, then this PR becomes wrong and should be closed (as detailed above and in earlier comments). This PR is unusual. Not only we need to review the proposed changes but _how_ those changes were accomplished. Only automated HERE usage removal changes are acceptable.

> go ahead with the automated parts first and manual fixes later.

That is exactly right, and this PR does the first "automated part" step. The manual changes will continue as usual, when those debugs() lines are changed for a good reason.

I do not understand why you insist that some of the automated HERE removal changes are excluded. They are not harmful. They do not block progress. They are very helpful to those maintaining branches that cannot closely track master. I see no valid reason to block them. The only implicit reason you gave so far (AFAICT) is that they do not change enough. That is not a valid reason to block them because "changing enough" automatically is impossible and without automation this PR should be rejected.

> Or we can focus on doing the manual changes first as much smaller cleanup patch(es)

This strategy has already been discussed and rejected: If you want to polish debugs() manually, wait for those debugs() to change for some other reason. Minor polishing does not justify creating painful conflicts.

While automated polishing may also create conflicts, they are much less painful.